client/landing/gutenboarding/variables.scss

```
@@ -1,4 +1,5 @@
 // Reusable style variables
+$gutenboarding-footer-height: 64px;
```

```suggestion
$gutenboarding-footer-height: 62px;
```

Just to _really_ nit pick 😆 
This makes the padding perfectly match the mobile footer mockup in this comment https://github.com/Automattic/wp-calypso/pull/41731#pullrequestreview-405722783
Unless you followed a different mockup, in which case we've got a mockup war on our hands!

---

I appreciate the nitpick 😃 Let's leave it at 64px though as I design everything on a 8px grid. There might be smaller inconsistencies in the mocks, but overall spacing will look better if we stick to 8px increments.